### PR TITLE
Add dev.edfloreshz.Done.Devel

### PR DESCRIPTION
**Pull request by edfloreshz**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: dev.edfloreshz.Done.Devel.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                "unzip protobuf.zip -d protobuf",
+                "mkdir -p $FLATPAK_DEST/bin/",
+                "chmod 755 protobuf/bin/protoc",
+                "cp protobuf/bin/protoc $FLATPAK_DEST/bin/",
+                "cp -dr protobuf/include $C_INCLUDE_PATH/"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/protocolbuffers/protobuf/releases/download/v21.12/protoc-21.12-linux-x86_64.zip",
```
Comment (hfiguiere) [inline dev.edfloreshz.Done.Devel.json:48]: how will this work on aarch64?

File: dev.edfloreshz.Done.Devel.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            "build-commands": [
+                "ar x libprotobuf-dev.deb",
+                "tar -xf data.tar.xz",
+                "mkdir -p $FLATPAK_DEST/lib/pkgconfig",
+                "cp -dr usr/lib/x86_64-linux-gnu/pkgconfig/* $FLATPAK_DEST/lib/pkgconfig"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "http://ftp.us.debian.org/debian/pool/main/p/protobuf/libprotobuf-dev_3.21.12-1+b1_amd64.deb",
```
Comment (hfiguiere) [inline dev.edfloreshz.Done.Devel.json:66]: how will this work on aarch64.

You should build it from source.

File: dev.edfloreshz.Done.Devel.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                }
+            ]
+        },
+        {
+            "name" : "done",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dbuildtype=release",
+                "-Dprofile=development"
```
Comment (hfiguiere) [inline dev.edfloreshz.Done.Devel.json:78]: I doubt this should be development profile....

Comment (edfloreshz) [reply dev.edfloreshz.Done.Devel.json:78]: This is the development version of the app.

File: dev.edfloreshz.Done.Devel.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
+{
+    "app-id" : "dev.edfloreshz.Done.Devel",
```
Comment (hfiguiere) [inline dev.edfloreshz.Done.Devel.json:2]: In general Flathub is not for development snapshots,

Comment (edfloreshz) [reply dev.edfloreshz.Done.Devel.json:2]: Oh okay, sorry about that, I saw some apps on the repo ending in `.Devel` and wanted to upload my app as well.

Comment (hfiguiere) [reply dev.edfloreshz.Done.Devel.json:2]: he very few are EXCEPTIONS.

Comment (edfloreshz) [reply dev.edfloreshz.Done.Devel.json:2]: In what cases are these exceptions made?

Comment (barthalion) [reply dev.edfloreshz.Done.Devel.json:2]: We accepted a single `Devel` app. Flathub is generally meant for stable releases, developers can also use `beta` branch for testing.

**Pull request closed without merging**